pages/common/moe.md

```diff
@@ -0,0 +1,20 @@
+# moe
+
+> A WYSIWYG text editor for ISO-8859-15 encoded text.
+> More information: <https://www.gnu.org/software/moe/moe.html>.
+
+- Edit a file:
+
+`moe {{path/to/file}}`
+
+- Save and Quit:
```

Comment:
Please move the hotkeys at the end, first the command examples.

Reply:
is this mentioned in the content guidelines or style guide? or is that just another informal rule you have?

Reply:
and also: shouldn't `tldr vim` then also be changed, because `:help` and `:wq` and listed before another command?